examples/examples.py

```diff
@@ -91,6 +91,7 @@
     },
     "large_2019_rl": {"scenario": "example_03a", "study_case": "base_case_2019"},
     "large_2021_rl": {"scenario": "example_03b", "study_case": "base_case_2021"},
+    "experiment": {"scenario": "experiment", "study_case": "dst_flexibility_case"},
```

Comment:
experiment is not the best name for the example. maybe use the example_01 form or add the data to one of the existing examples? for example from your tutorial? I believe we don't need this extra folder